_posts/2019-10-20-iPhone-11.md

```diff
@@ -0,0 +1,45 @@
+# iPhone 11
+
+I've now spent one month using iPhone 11 as my daily driver.
+
+It is a big step up from my previous 250 EUR budget Android phone and my 5 year old iPhone 5S. This article should be seen in that light. I hadn't try a modern iOS device as my daily driver before so a lot of things that might be obvious to people using an iPhone X are not to me and I'm sure some things I point out were there in previous iPhones as well.
+
+## Pros
+
+- Great Camera (even in low light conditions)
+With both my Android phone and the 5S I had to use my wife's phone or my reflex to be able to get a non grainy picture on any light condition that was less than perfect. With iPhone 11 I don't need to worry, even at night your pictures will be more than acceptable.
```

Comment:
Windows Phone lol Carl Zeiss ftw

on - > in any light condition

your - > the (is is a personal review)

Reply:
sticking with "you"